modules/assim.sequential/tests/testthat/test_downscaling.R

```diff
@@ -0,0 +1,79 @@
+library(testthat)
+library(sf)
+library(terra)
+library(randomForest)
+library(withr)
```

Comment:
Consider removing these -- testthat gets loaded by the package-level testthat.R, and since we want package functions to behave right when a dependency isn't attached, I think there's virtue in making ourselves run the tests without attaching things.